projects/spectator/jest/src/lib/matchers-types.ts

```diff
@@ -34,6 +34,8 @@ declare namespace jest {
 
     toBeEmpty(): boolean;
 
+    toBePartial(partial: any): boolean;
```

Comment:
Why any? It should a least be an object.

Reply:
You are right! Fixed.